announcements/1.13.1.md

```diff
@@ -0,0 +1,35 @@
+### Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.13.1.
+
+Please see [component versions](https://anbox-cloud.io/docs/component-versions) for a list of updated components.
+
+#### New features & improvements
+
+ * Android security updates for January 2022 (see [Android Security Bulletin - March 2022](https://source.android.com/security/bulletin/2022-03-01) for more information)
+ * Android WebView has been updated to [99.0.4844.58](https://chromereleases.googleblog.com/2022/03/stable-channel-update-for-desktop.html)
```

Comment:
The link says 99.0.4844.51, but we might be using a newer version?

Reply:
There is no blog post for .58 but .51 and .58 are from the same build, just different revs for different platforms (Desktop, Android).